src/cardTypes.js

```diff
@@ -0,0 +1,35 @@
+export const dankort = {
```

Comment:
I created these 3 definitions here while they aren't supported by [credit-card-type](https://github.com/braintree/credit-card-type)

Reply:
As we discussed, use this file (with a different name) to abstract the usage of `credit-card-type` and `luhn` so it will be easy to replace it in the future if we need to.

Reply:
I moved this file to `/utils`.

Initially, I had the helpers and these objects in the same file, but it was a bit confusing for me, so I decided to split into 2 files just to be easier to understand what's being done in each of them.

Let me know what do you think.